pydrive2/test/settings/default.yaml

```diff
@@ -1,6 +1,6 @@
 client_config_backend: service
 service_config:
-  client_json_file_path: /tmp/pydrive2/credentials.json
```

Comment:
let's keep it as it was before please, unless it's important in this case (could you clarify the change then?)